project/dimensions/subsectors.csv

```diff
@@ -0,0 +1,38 @@
+id,name
+unspecified,Unspecified
+bev_compact,BEV Compact
+bev_midsize,BEV Midsize
+bev_pickup,BEV Pickup
+bev_suv,BEV SUV
+phev_compact,PHEV Compact
+phev_midsize,PHEV Midsize
+phev_pickup,PHEV Pickup
+phev_suv,PHEV SUV
+bev_light_medium,BEV Light-Medium
+bev_medium,BEV Medium
+bev_non_freight_truck,BEV Non-freight truck
+bev_heavy,BEV Heavy
+bev_school_bus,BEV School Bus
+bev_transit_bus,BEV Transit Bus
+light_rail,Light Rail
+naics_311,Food Manufacturing
+naics_312,Beverage and Tobacco Product Manufacturing
+naics_313,Textile Mills
+naics_314,Textile Product Mills
+naics_315,Apparel Manufacturing
+naics_316,Leather and Allied Product Manufacturing
+naics_321,Wood Product Manufacturing
+naics_322,Paper Manufacturing
+naics_323,Printing and Related Support Activities
+naics_324,Petroleum and Coal Products Manufacturing
+naics_325,Chemical Manufacturing
+naics_326,Plastics and Rubber Products Manufacturing
+naics_327,Nonmetallic Mineral Product Manufacturing
+naics_331,Primary Metal Manufacturing
+naics_332,Fabricated Metal Product Manufacturing
+naics_333,Machinery Manufacturing
+naics_334,Computer and Electronic Product Manufacturing
+naics_335,"Electrical Equipment, Appliance, and Component Manufacturing"
+naics_336,Transportation Equipment Manufacturing
+naics_337,Furniture and Related Product Manufacturing
+naics_339,Miscellaneous Manufacturing
```

Comment:
Industry, please revise. I started here, with the 3-digit manufacturing NAICS codes, based on the DECARB Task 6 approach document.